src/main/java/edu/northeastern/truthtree/AppConst.java

```diff
@@ -51,7 +52,7 @@
 
         public static final String TIME_RANGE_FILE_PATH = RESOURCES_PATH + "/TimeRange.json";
 
-	// Population Adapter Constants
-	public static final String POPULATION_SPEC_PATH = RESOURCES_PATH + "/PopulationSpec.json";
+	    // Population Adapter Constants
```

Comment:
It's better to keep the format consistent

Reply:
@peishanwang Just made the changes as per your suggestion. 
I am not sure why an extra tab space was added to it.